docs/extend/plugins_graphdriver.md

```diff
@@ -34,8 +58,18 @@ Plugins are not required to accept these options as the Docker Engine does not
 require that the plugin use this path or options, they are only being passed
 through from the user.
 
-**Response**:
+A list of UID and GID mappings used for user namespaces is also provided. These
```

Comment:
How about something like:

The response also includes a list of UID and GID mappings, structured as follows: